api/src/main/java/marquez/graphql/MarquezGraphqlServletBuilder.java

```diff
@@ -0,0 +1,226 @@
+package marquez.graphql;
+
+import static graphql.schema.idl.TypeRuntimeWiring.newTypeWiring;
+
+import com.codahale.metrics.MetricRegistry;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.google.common.base.Charsets;
+import com.google.common.io.Resources;
+import com.smoketurner.dropwizard.graphql.CachingPreparsedDocumentProvider;
+import com.smoketurner.dropwizard.graphql.GraphQLFactory;
+import graphql.execution.preparsed.PreparsedDocumentProvider;
+import graphql.kickstart.execution.GraphQLQueryInvoker;
+import graphql.kickstart.servlet.GraphQLHttpServlet;
+import graphql.schema.Coercing;
+import graphql.schema.CoercingParseLiteralException;
+import graphql.schema.CoercingParseValueException;
+import graphql.schema.CoercingSerializeException;
+import graphql.schema.GraphQLScalarType;
+import graphql.schema.GraphQLSchema;
+import graphql.schema.idl.RuntimeWiring;
+import graphql.schema.idl.SchemaGenerator;
+import graphql.schema.idl.SchemaParser;
+import graphql.schema.idl.TypeDefinitionRegistry;
+import java.net.URL;
+import java.time.ZonedDateTime;
+import java.util.Map;
+import java.util.UUID;
+import lombok.SneakyThrows;
+import org.jdbi.v3.core.Jdbi;
+
+public class MarquezGraphqlServletBuilder {
+
+  public GraphQLHttpServlet getServlet(final Jdbi jdbi) {
+    final GraphQLFactory factory = getGraphQLFactory(jdbi);
+
+    final PreparsedDocumentProvider provider =
+        new CachingPreparsedDocumentProvider(factory.getQueryCache(), new MetricRegistry());
```

Comment:
Does this get registered with the global metric registry? You may have to pass in `Environment` and access the metric registry via `env.metrics()` see, https://github.com/MarquezProject/marquez/blob/main/api/src/main/java/marquez/MarquezApp.java#L80